cc/gtp_player.cc

```diff
@@ -122,6 +124,21 @@ GtpPlayer::Response GtpPlayer::DispatchCmd(
   return Response::Error("unknown command");
 }
 
+GtpPlayer::Response GtpPlayer::HandleListCommands(
+    absl::string_view cmd, const std::vector<absl::string_view>& args) {
+  auto response = CheckArgsExact(cmd, 0, args);
+  if (!response.ok) {
+    return response;
+  }
+  absl::string_view commands[] = {
```

Comment:
can you add a TODO to put the command handlers in a map<string, std::function> so we can avoid having to list the commands separately here?